### ISSUE TITLE
remove white space character

### ISSUE THREAD

**Issue by bouzuya** [b-html/b-html#24]
Current:

```
<p>
  <a
    href="#"
    style="border: 1px solid #f00"
    >
    foo
  </a>
</p>
```

result: `|foo |` (has a blank)

I want:

```
<p>
  <a
    href="#"
    style="border: 1px solid #f00"
    >foo</a>
</p>
```

result: `|foo|` (has no blank)

**Comment by bouzuya** [comment]
```
<!DOCTYPE html>
<html
  lang="en"
  >
  <head>
    <meta
      charset="utf-8"
      />
    <title>
      b-html demo</title></head>
  <body>
    <h1>
      b-html demo</h1>
    <p>
      Source Code:
      <a
        href="https://github.com/b-html/b-html"
        >
        b-html/b-html</a></p></body></html>
```

**Comment by bouzuya** [comment]
NOTE: add `demo` option.

**Comment by bouzuya** [comment]
Fixed: #26 & #27

**Comment by bouzuya** [comment]
0.7.0
